argus/scenarios/service_mock.py

```diff
@@ -183,7 +183,7 @@ def __init__(self, scenario):
 
     @cherrypy.expose
     def index(self):
-        expected_header = "Domu_request"
+        expected_header = "Domu-Request"
```

Comment:
Why this fix? https://review.openstack.org/#/c/149089/13/cloudbaseinit/metadata/services/cloudstack.py seems to suggest that DomU_Request is the correct header.